### DISCUSSION TITLE
add "start terminal" command to right-click context menu

### DISCUSSION THREAD

**Discussion by capaj** [spacedriveapp/spacedrive#2759] [Feature Request]
Context menu when I click on empty space currently has these two options:

![image](https://github.com/user-attachments/assets/1b308c4b-60ea-4d6d-83b4-2b2464b59132)

would be nice if it had option to start a terminal session in that folder

**Comment by Frank01001** [comment]
This is a must for any Linux user. I intend to replace Nautilus with Spacedrive on my system, but not having "open in terminal" is a dealbreaker for me.

**Comment by HeavenVolkoff** [comment]
Hey, I added an issue for this discussion:
https://github.com/spacedriveapp/spacedrive/issues/2787

If anyone wants to implement this, feel free, it is otherwise a low priority issue for the team atm (we are focused on the v0.5.0 release)
